### PR TITLE
[AC-4670e-api] - BLOCKED - Switch Currency Model to django-accelerator

### PR DESCRIPTION
**Pull request by yotammanor**
**Changes Introduced in [AC-4670](https://masschallenge.atlassian.net/browse/AC-4670)**

- replace accelerate's Currency model with django-accelerator's Currency model.
- follow the Green/Blue process as suggested in [AC-4308](https://masschallenge.atlassian.net/browse/AC-4308).
- This is the impact-api side, following in tandem with accelerate changes

**Test:**
- on accelerate, checkout AC-4670e, pull, resetdb, dumpdb.
- on impact-api, checkout AC-4670e-api, pull, make build, make dev.
- separate window, make dbload (the newly dumped db).
- `make grant-permissions PERMISSION_USER=demoadmin@masschallenge.org PERMISSION_CLASSES=v0_clients,v1_clients`
- all api operations on startup (as per [collection](https://www.getpostman.com/collections/e20616ae9b0b627db313) should still work (see setup instructions in #21). 
- Note that new_currency field is no longer available, only currency.
- Note that http://localhost:8000/api/impact/Currency/ is no longer available.
  

**Note**
- Step 6 will be skipped for impact-api, so it's worth repeating steps above with a dbdump from AC-4670f (which should work just the same).

### CODE REVIEW

File: web/impact/impact/settings.py [hunk 1]
```diff
@@ -79,7 +79,9 @@ class Base(Configuration):
         'oauth2_provider',
         'corsheaders',
         'embed_video',
+        'accelerator.apps.AcceleratorConfig',
```
Comment (pbx) [inline web/impact/impact/settings.py:82]: What is this about?

**Review by pbx — COMMENTED** [review]
Works as it should, I believe. 

When I ran `make dev` I did notice a "models have changes that are not yet reflected in a migration" notice fly by (after having `make dbload`ed the `dumpdb` output from accelerate@AC-4670e). Expected?

**Review by mc-nathan — APPROVED** [review]
Looks good.

File: web/impact/impact/settings.py (hunk 1 above)
Comment (mc-nathan) [reply web/impact/impact/settings.py:82]: It's the name of the django-accelerator app.

**Pull request merged**